apps/hash-frontend/src/shared/layout/layout-with-sidebar/account-entity-type-list/entity-type-item.tsx

```diff
@@ -69,6 +69,11 @@ export const EntityTypeItem: FunctionComponent<EntityTypeItemProps> = ({
     router.route === "/[shortname]/types/entity-type/[entity-type-id]" &&
     urlBase === baseUrl;
 
+  const { isLink } = useIsSpecialEntityType({
+    allOf,
+    $id: entityTypeId,
+  });
```

Comment:
I think you can use `isSpecialEntityTypeLookup` here instead, since the entity is already in the db

Reply:
Thanks for flagging, have addressed in https://github.com/hashintel/hash/pull/3610/commits/b0f13528ea01f7dda67ad029ee6eab6f11b4036b